data/transactions/logic/evalStateful_test.go

```diff
@@ -461,13 +479,20 @@ func testAppsBytes(t *testing.T, programs [][]byte, ep *EvalParams, expected ...
 
 func testApp(t *testing.T, program string, ep *EvalParams, problems ...string) transactions.EvalDelta {
 	t.Helper()
+	if ep == nil {
+		ep = defaultAppParamsWithVersion(LogicVersion)
+	}
```

Comment:
Should this call `ep.reset()` if it's not nil? This function appears to be called multiple times with the same `ep` on different programs.

Reply:
nvm, this just calls into `testAppBytes`, which does reset the `ep`